app/forms/state_file/name_dob_form.rb

```diff
@@ -1,127 +0,0 @@
-module StateFile
-  class NameDobForm < QuestionsForm
-    include DateHelper
-
-    attr_accessor :dependents_attributes
-
-    set_attributes_for :intake,
-                       :primary_first_name,
-                       :primary_middle_initial,
-                       :primary_last_name,
-                       :primary_suffix,
-                       :spouse_first_name,
-                       :spouse_middle_initial,
-                       :spouse_last_name,
-                       :spouse_suffix,
-                       :primary_birth_date_month, :primary_birth_date_day, :primary_birth_date_year,
-                       :spouse_birth_date_month, :spouse_birth_date_day, :spouse_birth_date_year
-    set_attributes_for :state_file_efile_device_info, :device_id
-
-
-    delegate :ask_months_in_home?,
-             :ask_spouse_dob?,
-             :ask_spouse_name?,
-             :filing_status_mfj?,
-             :hoh_qualifying_person_name,
-             to: :intake
-
-    validates_presence_of :primary_first_name, :primary_last_name
-    validates_presence_of :spouse_first_name, :spouse_last_name, if: -> { @intake.ask_spouse_name? }
-    validates_presence_of :hoh_qualifying_person_name, if: -> {  @intake.class == StateFileAzIntake && @intake.requires_hoh_qualifying_person_name? }
-    validate :primary_birth_date_is_valid_date
-    validate :spouse_birth_date_is_valid_date, if: -> { @intake.ask_spouse_dob? }
-    validates :primary_first_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z\-\s']{0,15})\z/.freeze, message: ->(_object, _data) { I18n.t('errors.attributes.first_name.invalid_format') }}
-    validates :primary_middle_initial, length: { maximum: 1 }, format: { with: /\A[A-Za-z]\z/.freeze, allow_blank: true }
-    validates :primary_last_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z\-\s']{0,137})\z/.freeze, message: ->(_object, _data) { I18n.t('errors.attributes.last_name.invalid_format') }}
-    validates :spouse_first_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z\-\s']{0,15})\z/.freeze, message: ->(_object, _data) { I18n.t('errors.attributes.first_name.invalid_format') }}, if: -> { @intake.ask_spouse_name? }
-    validates :spouse_middle_initial,  length: { maximum: 1 }, format: { with: /\A[A-Za-z]\z/.freeze, allow_blank: true }, if: -> { @intake.ask_spouse_name? }
-    validates :spouse_last_name, format: { with: /\A[a-zA-Z]{1}([A-Za-z\-\s']{0,137})\z/.freeze, message: ->(_object, _data) { I18n.t('errors.attributes.last_name.invalid_format') }}, if: -> { @intake.ask_spouse_name? }
```

Comment:
bunch of validations that were previously being enforced is no longer active -- this is fine b/c we're just trusting DF json data right?

Reply:
IMO yes, we shouldn't throw validation errors on information we don't collect from the user but get directly from DF. if there are incompatibilities between this information and our output schemas, those should be addressed (as best as possible) with output filtering/truncation, which I believe is already in place